sn_node_manager/src/bin/cli/main.rs

```diff
@@ -284,11 +284,22 @@ pub enum SubCmd {
     /// sudo if you defined system-wide services; otherwise, do not run the command elevated.
     #[clap(name = "start")]
     Start {
+        /// The max time in milliseconds to wait for the RPC connection to the node to be established.
+        ///
+        /// Setting this will automatically scale the interval between each service launch. This argument
+        /// is mutually exclusive with the 'interval' argument.
+        ///
+        /// The upgrade will fail if the connection is not established within this time.
+        #[clap(long, default_value = "120000", conflicts_with = "interval")]
+        connection_timeout: u64,
         /// An interval applied between launching each service.
         ///
+        /// Use connection-timeout to scale the interval automatically. This argument is mutually exclusive with the
```

Comment:
Considering my comment above, I don't think these arguments should really relate to one another, because I don't think the dynamic scaling should be tied to the `connection-timeout` argument.